consts/media_type.py

```diff
@@ -11,6 +11,7 @@ class MediaType(object):
     PERISCOPE_PROFILE = 8
     GRABCAD = 9
     INSTAGRAM_IMAGE = 10
+    FIRST_RESOURCE = 11
```

Comment:
Can we generalize this a little more to be any external link? That way, we can potentially reuse this type if we choose to allow teams to link their own essays, etc.